tests/ui/pattern/deref-patterns/euv-ice-125059.rs

```diff
@@ -1,6 +1,8 @@
-//@ known-bug: rust-lang/rust#125059
+//@ run-pass
```

Comment:
```suggestion
//@ check-pass
```

Given that this doesn't actually execute anything meaningful